MasterSeed32.md

```diff
@@ -1,11 +1,26 @@
-# Master Seed 32
+# codex32
 
 WARNING: Seriously, this is a work in progress, and it is only a concept right now.
 If you try to use this for your valuable data, I promise you will lose your data.
 You will lose this document and come back here only to find that I have made incompatible changes,
 and your data is lost forever. Even if you don't lose this document, there is no warranty or
 guarantee of any kind that you will be able to successfully recover your data.
 
+```
+  BIP: ????
+  Layer: Applications
+  Title: codex32
+  Author: <list of authors' real names and email addrs>
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-????
+  Status: Proposed
```

Comment:
Maybe start with draft until the analysis of the generators is completed?

Reply:
Actually I think BIP2 requires a full implementation somewhere before we can move to Proposed, and the postscript doesn't implement the long strings.

Reply:
It says that a lack of reference implementation won't block getting a number, so no need to prioritize this .. but I'll put together a full Python impl from the existing reference code and sipa's pychar stuff.